sdcm/utils/properties.py

```diff
@@ -0,0 +1,50 @@
+import io
+from typing import Union, TextIO
+
+
+class PropertiesDict(dict):
+    all_items = dict.items
+    all_keys = dict.keys
+    all_values = dict.values
+
+    def items(self):
+        for key, value in super().items():
+            if key.lstrip()[0] != '#':
+                yield key, value
+
+    def keys(self):
+        for key in super().keys():
+            if key.lstrip()[0] != '#':
+                yield key
```

Comment:
These 2 are unused. Why do we need these methods? We should not introduced unused/dead code "just for future".

Reply:
It is part of dict specification, you can push half-implemented classes when mimic something